framework/python/src/common/session.py

```diff
@@ -37,13 +37,39 @@
 MAX_DEVICE_REPORTS_KEY = 'max_device_reports'
 CERTS_PATH = 'local/root_certs'
 CONFIG_FILE_PATH = 'local/system.json'
+STATUS_TOPIC = 'status'
 
 PROFILE_FORMAT_PATH = 'resources/risk_assessment.json'
 PROFILES_DIR = 'local/risk_profiles'
 
 LOGGER = logger.get_logger('session')
 
 
+def session_tracker(method):
+  """Session changes tracker."""
+  def wrapper(self, *args, **kwargs):
+
+    result = method(self, *args, **kwargs)
+
+    if self.get_status() != 'Idle':
+      self.get_mqtt_client().send_message(STATUS_TOPIC, self.to_json())

```

Comment:
The JSON format is not correct. fastapi automatically converts some Python objects into JSON for us. Now that it is not used, raw Python objects are being included in the response. See here: https://paste.googleplex.com/6069255923826688 Before this can be merged, more work is required to verify the JSON being received by the UI.

Reply:
I fixed it in the ui branch